CHANGELOG.md

```diff
@@ -8,13 +8,21 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
-### Changed
+### Added
+
+- Add template and styling for persons list page,
```

Comment:
I don't see anything related to persons in this PR?

Reply:
As said before ( https://github.com/openfun/richie/pull/636#issuecomment-482394215 ), i added missing change log from already merged PR for person list.